src/routes/AppContent/index.tsx

```diff
@@ -70,6 +71,13 @@ const AppContent = () => {
         setActiveTab(getActiveTab(location.pathname));
     }, [location]);
 
+    useEffect(() => {
```

Comment:
[M] let's use ref here to track if we have allowed, its more lightweight and usable for this use case

Reply:
done 👍